### ISSUE TITLE
Document OSGi special case: Karaf and explain how to use Karaf "features"

### ISSUE THREAD

**Issue by denis-anisimov** [vaadin/docs#161]
Once this https://github.com/vaadin/platform/pull/1818 is merged the OSGi documentation should be extended with:
- mention Karaf (PAX-web) as a special OSGi case with specific `OSGiVaadinServlet` .
- mention Karaf base starter
- mention Vaadin Karaf "features": `flow` , `flow-osgi` , `vaadin-core`, `vaadin`.

**Comment by denis-anisimov** [comment]
This is already done.
